cluster/images/etcd/Makefile

```diff
@@ -49,6 +49,12 @@ PUSH_REGISTRY?=staging-k8s.gcr.io
 
 MANIFEST_IMAGE := $(PUSH_REGISTRY)/etcd
 
+SELINUX_ENABLED := $(shell cat /sys/fs/selinux/enforce)
```

Comment:
these images can otherwise also be built from mac AFAIK, can we do:
```suggestion
SELINUX_ENABLED := $(shell cat /sys/fs/selinux/enforce || echo 0)
```

Reply:
Would still have worked, but this is admittedly explicit and cleaner.

Reply:
yeah, it's still slightly annoying because the stderr comes out, not sure how to cleanly deal with that here. `cat: /sys/fs/selinux/enforce: No such file or directory`

Reply:
🤷‍♂

Reply:
I did intend to check for that but didn't spot it when I was testing. I've redirected stderr to /dev/null now